sdk/nginx/ci.yml

```diff
@@ -0,0 +1,32 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+    - main
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/nginx/
+
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/nginx/
+    - sdk/resourcemanager/
```

Comment:
FYI @weshaggard does this need to get sucked in to the mgmt pipeline optimizations/yml edits?

Reply:
Yes please run https://github.com/Azure/azure-sdk-for-net/blob/main/eng/scripts/Update-Mgmt-CI.ps1 and it will correctly fix this ci.yml and add it to resourcemanager.

Reply:
Done